Library/Tracking/Koala.swift

```diff
@@ -701,6 +713,24 @@ public final class Koala {
     )
   }
 
+  public func trackPledgeSubmitButtonClicked(
```

Comment:
Was this added as part of this ticket? I thought we already have a `Pledge Submit Button Clicked` event 🤔

Reply:
It's on master: https://github.com/kickstarter/ios-oss/blob/master/Library/Tracking/Koala.swift#L680